mod/control.lua

```diff
@@ -192,7 +191,7 @@ function gen_surface_info(params, surface)
       chunk_count = chunk_count + 1
 
       if try_ent_only then
-        in_ent = surface.count_entities_filtered({ area = chunk.area, limit = 1, type = entities.includes}) > 0
+        in_ent = surface.count_entities_filtered({ area = chunk.area, limit = 1, force = 'player'}) > 0
```

Comment:
As you mention, that will change the behavior for existing multiplayer cases potentially. I don't have a good idea of how much it is used, but I would prefer to not break the behavior if not needed. I think the easiest way forward would be to add an extra way of selecting the area. Given the settings already existins, I think it can be straightforward:

 - In settings.lua, extending the `area` settings to have `allowed_values` set to something like `{"all", "entities", "player-force"}
 -Have `try_ent_only` set if `(params.area == "entities") || (params.area == "player-force")`
 - Here, within the `try_ent_only` if-block, have both variations of `count_entities_filtered` based on `params.area` value.

Reply:
You're welcome. Since I'm running this site, I'll be able to keep an eye on anything that comes up. I'm confident the switch is an improvement for the time being. Thanks for your great work.